### DISCUSSION TITLE
Can superset connect to iotdb?

### DISCUSSION THREAD

**Discussion by ixiaolang** [apache/superset#20548] [Q&A / Help]
Github Discussions is our new home for discussing features and improvements!

https://github.com/apache/superset/discussions/categories/ideas

We'd like to keep Github Issues focuses on bugs and SIP's (Superset Improvement Proposals)!

Please note that feature requests opened as Github Issues will be moved to Discussions.

**Comment by srinify** [comment]
Hey there, it looks like iotdb may not have a sqlalchemy dialect. This is a requirement for Superset at the moment. There's a great response here:

https://lists.apache.org/thread/mdnvbzg4l31wp92tm40mrs7x013h5z0d
